idaes/generic_models/unit_models/column_models/column.py

```diff
@@ -0,0 +1,1144 @@
+                return Ackmann_factor /\
+                    (1 - exp(-Ackmann_factor /
+                             (blk.h_v[t, x] * blk.area_interfacial[t, x] *
+                              blk.area_column)))
+        self.h_v_Ack = Expression(
+            self.flowsheet().time,
+            self.vapor_phase.length_domain,
+            rule=rule_heat_transfer_coeff_Ack,
+            doc='Vap-Liq heat transfer coefficient corrected by Ackmann factor')
+
+        # Heat transfer  - vapor side [J/s.m]
+        @self.Constraint(self.flowsheet().time,
+                         self.vapor_phase.length_domain,
+                         doc="heat transfer - vapor side ")
+        def vapor_phase_heat_transfer(blk, t, x):
+            if x == self.vapor_phase.length_domain.first():
+                return blk.heat_transfer_rate_vap[t, x] == 0
+            else:
+                zb = self.vapor_phase.length_domain.at(value(self.zi[x]) - 1)
+                return blk.heat_transfer_rate_vap[t, x] == blk.h_v_Ack[t, x] * \
+                    (blk.liquid_phase.properties[t, zb].temperature -
+                     blk.vapor_phase.properties[t, x].temperature)
+
+        # Heat transfer - liquid side [J/s.m]
+        @self.Constraint(self.flowsheet().time,
+                         self.liquid_phase.length_domain,
+                         doc="heat transfer - liquid side ")
+        def liquid_phase_heat_transfer(blk, t, x):
+            if x == self.liquid_phase.length_domain.last():
+                return blk.heat_transfer_rate_liq[t, x] == 0
+            else:
+                zf = self.vapor_phase.length_domain.at(value(self.zi[x]) + 1)
+                return blk.heat_transfer_rate_liq[t, x] == blk.heat_transfer_rate_vap[t, zf] + \
+                    (sum(blk.liquid_phase.properties[t, x].enth_mol_phase_comp['Liq',j] *
+                     blk.interphase_mass_transfer[t, zf, j] for j in solute_comp_list) -
+                     sum(blk.liquid_phase.properties[t, x].enth_mol_phase_comp['Liq',k] *
+                     blk.interphase_mass_transfer[t, zf, k] for k in solvent_comp_list))
+
+        # Vapor phase heat transfer handle
+        @self.Constraint(self.flowsheet().time,
+                         self.vapor_phase.length_domain,
+                         doc="vapor - heat transfer handle")
+        def vapor_phase_heat_transfer_handle(blk, t, x):
+            return blk.vapor_phase.heat[t, x] == blk.heat_transfer_rate_vap[t, x]
+
+        # Liquid phase heat transfer handle
+        @self.Constraint(self.flowsheet().time,
+                         self.liquid_phase.length_domain,
+                         doc="liquid - heat transfer handle")
+        def liquid_phase_heat_transfer_handle(blk, t, x):
+            return blk.liquid_phase.heat[t, x] == -blk.heat_transfer_rate_liq[t, x]
+
+        if self.config.dynamic:
+            self.fix_initial_condition()
```

Comment:
This should not be part of the unit model. This is something the user should do when initializing the model, rather than done automatically during model construction.